Demo-OSX.xcodeproj/project.pbxproj

```diff
@@ -283,7 +283,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		32488D9F2183500500179930 /* Mozilla_Firefox_logo_2013.svg */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; name = Mozilla_Firefox_logo_2013.svg; path = "../../../SVGKit-2.x/Demo-Samples/SVG/Mozilla_Firefox_logo_2013.svg"; sourceTree = "<group>"; };
```

Comment:
Strange, this image use a relative path and cause build fail on Travis-CI. Fix it to use the correct path